### ISSUE TITLE
ASSEMBLY 2024 PORTUGAL ISSUE CORRECTION

### ISSUE THREAD

**Issue by Visionario012** [yapms/YAPms#560]
Hello, Im portuguese and I want to congratulate the work that you people have been doing creating these election simulators. It's fantastic

I am not a coder but I humbly request for someone to correct the simulator of the PORTUGAL ASSEMBLY 2024. You only need to copy from the OLD YAPMS the way the simulator works. In each district, there are a number of delegates/deputys that are distributed to each political party depending on the vote they had in the disctrict. The election process uses D'Hondt Method in each district. Then, the sum of all the delegates/deputys won in each district by the parties will determine the winner. 

Basically, you only need to copy what has been done from the OLD YAPMS.

Cheers and keep up the good work

**Comment by Flizzyclone** [comment]
ill fix this at some point. to fix this you can switch Mode to SPLIT
![image](https://github.com/yapms/yapms/assets/42476312/28920807-acb1-473e-8836-337055fa8acf)

**Comment by Visionario012** [comment]
thanks
